json-rpc/src/stream_rpc/connection/client_connection.rs

```diff
@@ -115,14 +115,28 @@ impl ClientConnection {
     }
 
     /// Unsubscribe a subscription for a client
-    #[allow(unused)]
     pub fn unsubscribe(&self, id: &Id) -> Option<()> {
         let mut tasks = self.tasks.lock();
         if let Some(task) = tasks.get(id) {
             debug!(
                 "Unsubscribing: terminating task '{}' for Client#{}",
                 &id, &self.id
             );
+
+            // Send off a response to the client
+            let sender = self.sender.clone();
+            let id2 = id.clone();
+            tokio::spawn(async move {
+                let response = StreamJsonRpcResponse::result(
+                    Some(id2),
+                    Some(serde_json::to_value(UnsubscribeResult::ok()).unwrap()),
+                );
+                sender
+                    .send(Ok(serde_json::to_string(&response).unwrap()))
+                    .await
+                    .ok()
```

Comment:
Same thing here, we're ignoring the result from the send (which is ok) but it would be good to document that